content/docs/platform/inbox/configuration/tabs.mdx

```diff
@@ -4,28 +4,29 @@ description: 'Learn what tabs are and how to filter multiple tabs in the Novu In
 icon: 'Folders'
 ---
 
-Tabs in the Novu Inbox enable you to organize and display different sets of notifications within distinct UI tabs. This creates a structured, filterable interface for end users to easily navigate their in-app notification feed.
 
-Each tab is defined using the `tabs` prop of the <Method href="/platform/inbox/react/components/inbox">{`<Inbox />`}</Method> component. Every tab object include the following properties:
-- **`label`**: A string representing the tab's visible title.
-- **`filter`**: An object defining the filter criteria for notifications displayed in that tab.
-
-The `filter` property supports two filtering options:
-- **Tags**: Filter based on the `tags` assigned to a workflow in the workflow editor.
-- **Data attributes**: Filter based on values in the notification payload (`data`).
+Tabs in the Inbox component are used to organize in-app notifications into separate sections, creating a cleaner, more organized experience for your subscribers. Each tab displays a subset of in-app notifications based on defined rules, making it easier to surface important notifications.
 
 ![Filtering tabs by tags and data attributes](/images/inbox/filter-tabs-by-tags-and-data-attributes.png)
 
-## Filter tabs by tags
 
-Each notification is associated with a workflow, and workflows can be tagged in the workflow editor. These tags can then be used to group notifications from one or more workflows under a single tab by referencing shared tag names.
+## Group notifications into tabs
+
+Each tab is defined using the `tabs` prop of the Inbox component. Every tab object includes the following properties:
+- **`label`**: A string that defines the tab’s title, displayed as the visible label in the UI
+- **`filter`**: An object that defines which notifications are included in a particular tab.
+
+### Using tags
+
+Tags can be assigned to any workflow in the Novu workflow editor. These tags can then be used in the filter object to group notifications from one or more workflows under the same tab.
```

Comment:
@Aviatorscode2 

we should add workflow tags section link here, so user can read on how to create tags on the workflow